scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -126,6 +126,15 @@ class PublishVisibility(Enum):
     UNLISTED = 'unlisted'
 
 
+class SystemRegisty:
+    """Prod System registries where assets lie."""
+
+    AZUREML = "azureml"
+    AZUREML_META = "azureml-meta"
+    AZUREML_MSR = "azureml-msr"
+    NVIDIA_AI = "nvida-ai"

```

Comment:
Just wanted to check if this is a typo?
```suggestion
    NVIDIA_AI = "nvidia-ai"

```